include/WCSimWCDigi.hh

```diff
@@ -63,13 +65,14 @@ private:
    *  The second digit is made up of photons: 10,11,13,14
    */
   std::map<int, std::vector<int> > fDigiComp;
-  std::map<int, G4int>            trackID; // TrackID of the photon of the Hit (do not use for Digits)
-  std::map<int, G4int>    parentSavedTrackID; ///< Primary parent ID of the Hit (do not use for Digits)
-  std::map<int, G4float>    photonStartTime; ///< Primary parent ID of the Hit (do not use for Digits)
-  std::map<int, G4ThreeVector>    photonStartPos; ///< Start point of the photon of the Hit (do not use for Digits)
-  std::map<int, G4ThreeVector>    photonEndPos; ///< End point of the photon of the Hit (do not use for Digits)
-  std::map<int, G4ThreeVector>    photonStartDir; ///< Start dir of the photon of the Hit (do not use for Digits)
-  std::map<int, G4ThreeVector>    photonEndDir; ///< End dir of the photon of the Hit (do not use for Digits)
+  std::map<int, G4int>             trackID;              ///< TrackID of the photon of the Hit (do not use for Digits)
+  std::map<int, G4int>             parentSavedTrackID;   ///< Primary parent ID of the Hit (do not use for Digits)
+  std::map<int, G4float>           photonStartTime;      ///< Primary parent ID of the Hit (do not use for Digits)
+  std::map<int, G4ThreeVector>     photonStartPos;       ///< Start point of the photon of the Hit (do not use for Digits)
+  std::map<int, G4ThreeVector>     photonEndPos;         ///< End point of the photon of the Hit (do not use for Digits)
+  std::map<int, G4ThreeVector>     photonStartDir;       ///< Start dir of the photon of the Hit (do not use for Digits)
+  std::map<int, G4ThreeVector>     photonEndDir;         ///< End dir of the photon of the Hit (do not use for Digits)
+  std::map<int, std::string>       photonCreatorProcess; ///< Process which created the photon of the Hit (Diego Costas)
```

Comment:
you can already use `ProcessType_t` for this member

Reply:
👍